samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -48,6 +48,16 @@
     }
   },
 
+  "AzureSpeechConfig": {
+    "Label": "AzureSpeech",
+    "Region": "westus2", // Your Speech SDK subscription's region(e.g. westus2)
```

Comment:
Since the region will always be specific to the user, I recommend removing the default value here.